src/spi.rs

```diff
@@ -220,9 +220,9 @@ mod embedded_hal_impl {
                         };
                         transfers.push(SpidevTransfer::read_write(tx, buf));
                     }
-                    SpiOperation::DelayUs(us) => {
-                        let us = (*us).try_into().unwrap_or(u16::MAX);
-                        transfers.push(SpidevTransfer::delay(us));
+                    SpiOperation::DelayNs(ns) => {
+                        let ns = (*ns).try_into().unwrap_or(u16::MAX);
+                        transfers.push(SpidevTransfer::delay(ns));
```

Comment:
The nanoseconds need to be converted to microseconds here rounding up to at least 1 if not 0

Reply:
Whoops, of course. Fixed up.